app.py

```diff
@@ -65,6 +67,20 @@ def page_not_found(error):
     ), 404
 
 
+# Redirects
+# ===
+@app.route('/docs/<path:path>')
```

Comment:
Should it also catch `/docs/` without path?
There is a link to /docs in header navigation that currently is in 404.

But I guess it's currently handled on kubernates.

Reply:
Very true. I'll fix this.

Reply:
Fixed.